packages/php-wasm/node/src/test/php.spec.ts

```diff
@@ -51,182 +52,276 @@ describe.each(SupportedPHPVersions)('PHP %s', (phpVersion) => {
 			expect(result.text).toEqual('stdout: WordPress\n');
 		});
 
-		it('popen("cat", "w")', async () => {
-			const result = await php.run({
-				code: `<?php
-				$path = __DIR__;
-				$fp = popen("cat > out", "w");
-				fwrite($fp, "WordPress\n");
-				fclose($fp);
+		if (phpVersion == '8.2') {
+			it('popen("cat", "w")', async () => {
+				const result = await php.run({
+					code: `<?php
 
-				$fp = popen("cat out", "r");
-				echo 'stdout: ' . fread($fp, 1024);
-				fclose($fp);
-			`,
-			});
+                    $fp = popen("cat > out", "w");
+                    fwrite($fp, "WordPress\n");
+                    fclose($fp);
 
-			expect(result.text).toEqual('stdout: WordPress\n');
-		});
+                    sleep(1);
```

Comment:
Good spot, I just ran into the same thing, this test passes randomly if `cat > out` finishes before this point.